src/Microsoft.PowerShell.Commands.Management/Microsoft.PowerShell.Commands.Management.csproj

```diff
@@ -47,7 +47,7 @@
 
   <ItemGroup>
     <!-- the following package(s) are from https://github.com/dotnet/corefx -->
-    <PackageReference Include="System.ServiceProcess.ServiceController" Version="6.0.0-rc.1.21425.7" />
+    <PackageReference Include="System.ServiceProcess.ServiceController" Version="6.0.0-rc.1.21430.11" />
```

Comment:
Not for the PR. I see all .Net Runtime packages have the same version string - maybe it makes sense to define it as a const in .prop file to reduce file changes?

Reply:
It is not always guaranteed.